tests/test_utils/test_progressbar_rich.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import time
+
+import mmengine
+
+
+class TestRichProgressBar:
+
+    def test_start(self):
+        # single task
+        prog_bar = mmengine.RichProgressBar()
+        prog_bar.add_task(100)
+        assert len(prog_bar.bar.tasks) == 1
+        assert prog_bar.bar.tasks[0].total == 100
+        assert prog_bar.infinite is False
+        del prog_bar
```

Comment:
What happens if we don't explicitly delete prog_bar? One of the reasons for implementing `__del__` is so that users don't need to worry about manually collecting the RichProgressBar instance.